test/public/simulationPasses/overviewAnchoredSimulationPasses.test.js

```diff
@@ -0,0 +1,223 @@
+        const response = await goToPage(page, 'anchored-simulation-passes-overview', { queryParameters: { dataPassId: 3 } });
+
+        // We expect the page to return the correct status code, making sure the server is running properly
+        expect(response.status()).to.equal(200);
+
+        // We expect the page to return the correct title, making sure there isn't another server running on this port
+        const title = await page.title();
+        expect(title).to.equal('AliceO2 Bookkeeping');
+        const headerBreadcrumbs = await page.$$('h2');
+        expect(await headerBreadcrumbs[0].evaluate((element) => element.innerText)).to.be.equal('Anchored MC');
+        expect(await headerBreadcrumbs[1].evaluate((element) => element.innerText)).to.be.equal('LHC22a_apass1');
+    });
+
+    it('shows correct datatypes in respective columns', async () => {
+        // Expectations of header texts being of a certain datatype
+        const headerDatatypes = {
+            name: (name) => periodNameRegex.test(name),
+            year: (year) => !isNaN(year),
+            pwg: (pwg) => /PWG.+/.test(pwg),
+            requestedEventsCount: (requestedEventsCount) => !isNaN(requestedEventsCount),
+            generatedEventsCount: (generatedEventsCount) => !isNaN(generatedEventsCount),
+            outpuSize: (outpuSize) => !isNaN(outpuSize),
+        };
+
+        // We find the headers matching the datatype keys
+        await page.waitForSelector('th');
+        const headers = await page.$$('th');
+        const headerIndices = {};
+        for (const [index, header] of headers.entries()) {
+            const headerContent = await page.evaluate((element) => element.id, header);
+            const matchingDatatype = Object.keys(headerDatatypes).find((key) => headerContent === key);
+            if (matchingDatatype !== undefined) {
+                headerIndices[index] = matchingDatatype;
+            }
+        }
+
+        // We expect every value of a header matching a datatype key to actually be of that datatype
+
+        // Use the third row because it is where statistics are present
+        const firstRowCells = await page.$$('tr:nth-of-type(3) td');
+        for (const [index, cell] of firstRowCells.entries()) {
+            if (index in headerIndices) {
+                const cellContent = await page.evaluate((element) => element.innerText, cell);
+                const expectedDatatype = headerDatatypes[headerIndices[index]](cellContent);
+                expect(expectedDatatype).to.be.true;
+            }
+        }
+    });
+
+    it('Should display the correct items counter at the bottom of the page', async () => {
+        await goToPage(page, 'anchored-simulation-passes-overview', { queryParameters: { dataPassId: 3 } });
+        await page.waitForSelector('#firstRowIndex');
+
+        expect(await page.$eval('#firstRowIndex', (element) => parseInt(element.innerText, 10))).to.equal(1);
```

Comment:
use `expectInnerText` utility, no need for the parseInt

Reply:
right